### ISSUE TITLE
Complete Cluster Software Collection (Master Servers, Slave Servers, Id (or sync) Server)

### ISSUE THREAD

**Issue by Konard** [Konard/LinksPlatform#75]

**Comment by werkzeuge** [comment]
Напомни, зачем нужен IdServer?

**Comment by Konard** [comment]
Чтобы раздавать идентификаторы серверам. Чтобы у кластера было общее адресное пространство, но каждый сервер хранил свою часть.

**Comment by werkzeuge** [comment]
Допускаешь ли ты локализацию данных? То есть, аналог локальных переменных в базе? Или IdServer как раз говорит о том, что все связи в базе - всегда глобальны?

**Comment by Konard** [comment]
> IdServer как раз говорит о том, что все связи в базе - всегда глобальны?

Да. IdServer нужен, чтобы объединить несколько отдельных баз данных в одну единую.
